streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBWindowStore.java

```diff
@@ -47,13 +54,12 @@
 import java.util.SimpleTimeZone;
 import java.util.concurrent.ConcurrentHashMap;
 
-public class RocksDBWindowStore<K, V> implements WindowStore<K, V> {
+public class RocksDBWindowStore<K, V> implements WindowStore<K, V>, CanSendOldValues {
```

Comment:
I thought we are trying to wrap the store as `metered` -> `cached` -> `inner` store, like what we did for `RocksDBStore` with `CachingKeyValueStore`; why for windowed store we are leaking the caching mechanism into the underlying inner store directly then?